### ISSUE TITLE
Prop Snatcher frequently slows down, console fills with hitches

### ISSUE THREAD

**Issue by Fighterbuilder** [Foohy/jazztronauts#135]
I recently started replaying Jazztronauts and I noticed that the Prop Snatcher will often slow down and not snatch props within the aim cone as quickly as it should. Taking a look at the console, I see a lot of messages similar to the following:
`HITCHED: 0.050s
 0.050s @gamemodes/jazztronauts/gamemode/lib/sh_snatch.lua : 537
  (Recommended: Call 'Yield' more often to release timeslice to engine)`
The time after HITCHED generally ranges between 0.050s to 0.100s, and all of the messages point to line 537.

**Comment by Commenter25** [comment]
I also got this testing on [Prop Hunt: Amaya](https://steamcommunity.com/sharedfiles/filedetails/?id=355046292), while stealing things, I was getting it attributed to `sh_snatch.lua` on lines `475` `478` `537` `546` `629` `816` `820` and `841`

...but these seem like random arbitrary lines, some of them being commented out or completely blank. It seems like it may just be computer speed dependent lag?

**Comment by Fighterbuilder** [comment]
Yeah, that sounds plausible. My PC _is_ turning 10 this year, after all 😅

If I end up getting a new one at the end of the year like I plan to, I'll try again and see if I still get that issue.
